packages/nitro-protocol/contracts/CountingApp.sol

```diff
@@ -39,6 +39,7 @@ contract CountingApp is ForceMoveApp {
             appData(b.appData).counter == appData(a.appData).counter + 1,
             'CountingApp: Counter must be incremented'
         );
+        // Note this is gas inefficient, and inferior to _bytesEqual in use elsewhere
         require(
             keccak256(b.outcome) == keccak256(a.outcome),
```

Comment:
This is not the best way to compare bytes, but this code is only used in testing so I just left a comment.